baton/hub.go

```diff
@@ -2,9 +2,12 @@ package baton
 
 import (
 	"encoding/json"
-	"log"
+
+	"github.com/Sirupsen/logrus"
 )
 
+var log = logrus.WithField("module", "Hub")
```

Comment:
Does it makes sense to have a global variable for the logger? What do other people do for Logrus?